redirects.json

```diff
@@ -2045,12 +2033,6 @@
     "status":"301",
     "condition":null
   },
-  {
-    "source":"/lib-v1/auth/mfa/q/platform/flutter/",
-    "target":"/flutter/prev/build-a-backend/auth/manage-mfa/",
-    "status":"301",
-    "condition":null
-  },
```

Comment:
neither the source nor target content exist

Reply:
Ack need to update the meta filter